### PR TITLE
feature: update footer panel

### PR DESCRIPTION
**Pull request by gcor**

### CODE REVIEW

File: components/UI/ProjectsPanel/ProjectsPanel.tsx [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+    return (
+        <FocusLock disabled={!openMenu}>
+            <div
+                style={{ backgroundColor: theme.background, color: theme.color }}
+                className={classNames(styles.panel, {
+                    [styles.panel_active]: openMenu,
+                })}
+            >
+                <div onClick={close} className={styles.panel__overlay} />
```
Comment (a-chabin) [inline components/UI/ProjectsPanel/ProjectsPanel.tsx:80]: semantic police 👮‍♀️

Let's use `<button>`

File: components/UI/ProjectsPanel/ProjectsPanel.tsx [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+                        <div className={styles.panel__divider} />
+                    </div>
+                </div>
+                <div className={styles.panel__toggle}>
+                    <div
+                        tabIndex={0}
+                        role="button"
+                        onKeyDown={onSpace}
+                        className={styles.toggle}
+                        onClick={toggle}
```
Comment (a-chabin) [inline components/UI/ProjectsPanel/ProjectsPanel.tsx:115]: semantic police 👮‍♀️ 

Let's use `<button>` [1]

File: components/UI/ProjectsPanel/constants.ts [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    id: 'map',
+    fullTitle: 'Карта \n Екатеринбурга',
+    shortTitle: 'Карта',
+    link: 'https://ekaterinburg.design',
+    logo: mapIcon.src,
+};
+export const PROJECT_GUIDES = {
+    id: 'guides',
+    fullTitle: 'Руководство',
+    shortTitle: 'Руководство',
```
Comment (a-chabin) [inline components/UI/ProjectsPanel/constants.ts:17]: typo:

'Руководства'

File: components/UI/ProjectsPanel/constants.ts [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+import devIcon from './icons/dev_dark.svg';
+import mapIcon from './icons/map_dark.svg';
+import guidesIcon from './icons/guides_dark.svg';
+import transportIcon from './icons/transport_dark.svg';
+
+export const PROJECT_MAP = {
+    id: 'map',
+    fullTitle: 'Карта \n Екатеринбурга',
+    shortTitle: 'Карта',
+    link: 'https://ekaterinburg.design',
```
Comment (a-chabin) [inline components/UI/ProjectsPanel/constants.ts:11]: Let's add link to `https://map.ekaterinburg.io`

File: components/UI/ProjectsPanel/constants.ts [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    fullTitle: 'Руководство',
+    shortTitle: 'Руководство',
+    link: 'https://ekaterinburg.design',
+    logo: guidesIcon.src,
+};
+export const PROJECT_TRANSPORT = {
+    id: 'transport',
+    fullTitle: 'Транспорт',
+    shortTitle: 'Транспорт',
+    link: 'https://ekaterinburg.design',
```
Comment (a-chabin) [inline components/UI/ProjectsPanel/constants.ts:25]: `https://transport.ekaterinburg.io`

File: components/UI/ProjectsPanel/ProjectsPanel.tsx (hunk 1 above)
Comment (gcor) [reply components/UI/ProjectsPanel/ProjectsPanel.tsx:80]: its overlay, event material don't use button)

File: components/UI/ProjectsPanel/ProjectsPanel.tsx (hunk 2 above)
Comment (gcor) [reply components/UI/ProjectsPanel/ProjectsPanel.tsx:115]: img tag can't be inside the button. I used role button

Comment (gcor) [reply components/UI/ProjectsPanel/ProjectsPanel.tsx]: removed in new designs)

File: components/UI/ProjectsPanel/ProjectsPanel.tsx (hunk 1 above)
Comment (a-chabin) [reply components/UI/ProjectsPanel/ProjectsPanel.tsx:59]: ooops

**Comment by a-chabin** [comment]
Thank you!

**Pull request merged**